### DISCUSSION TITLE
Superset Charts are missing after ingestion

### DISCUSSION THREAD

**Discussion by ArcanElement** [open-metadata/OpenMetadata#10853] [Q&A]
I have an OpenMetadata instance (v 0.13.2) running on a Kubernetes cluster, along with an instance of Superset that is backed by a Postgresql database, and a mysql instance I'm using as a data source. I'm trying to set up ingestion to get the dashboard metadata from Superset. I have 1 dashboard with 1 chart (querying mysql), and I am able to get a record for the dashboard without a problem, but it does not include the chart. I'm using the SupersetAPIConnection in the service connection with debug turned on in the ingestion.

I've tried digging through the logs, and there aren't any errors. Unfortunately, the logs do not record the data coming back from Superset, but they do record the API calls sent out, and I was able to see them logged in the Superset pod as well. In particular, I see these API calls coming into Superset:
"GET /api/v1/dashboard/?q=(page:0,page_size:1) HTTP/1.1" 200 1094 "-" "python-requests/2.28.0"
"GET /api/v1/chart/?q=(page:0,page_size:1) HTTP/1.1"
"GET /api/v1/chart/?q=(page:0,page_size:25) HTTP/1.1" 200 1497 "-" "python-requests/2.28.0"
"GET /api/v1/dashboard/?q=(page:0,page_size:1) HTTP/1.1" 200 1094 "-" "python-requests/2.28.0"
"GET /api/v1/dashboard/?q=(page:0,page_size:25) HTTP/1.1" 200 1212 "-" "python-requests/2.28.0"

I can run those, and I am getting out the chart data from the chart API calls. Any ideas why it isn't showing up in the dashboard in OpenMetadata?

**Comment by harshach** [comment]
cc @OnkarVO7

**Comment by OnkarVO7** [comment]
Hi @ArcanElement if possible could you please share the response coming from the charts API?

**Comment by saleksandrov-nf** [comment]
Hi! It seems we have the same problem with Superset.
We use the latest version of Open Metadata (1.0.3) and Apache Superset v1.5.0

When setting up ingestion (Superset API connection, db provider, user/password correct), connection test is successful, ingestion is successful, there are no errors in debug logs, but no dashboards/charts are ingested.
The only suspicious thing in Superset action logs is the absence of user:

User	Action	dttm
None	DashboardRestApi.get_list	2023-06-08 13:26:09.220114
None	DashboardRestApi.get_list	2023-06-08 13:26:08.987437
None	ChartRestApi.get_list	2023-06-08 13:26:08.729043
None	ChartRestApi.get_list	2023-06-08 13:26:08.490589
None	ChartRestApi.get_list	2023-06-08 13:26:08.241083
None	DashboardRestApi.get_list	2023-06-08 13:26:07.997687
None	DashboardRestApi.get_list	2023-06-08 13:26:07.768852

Maybe it's the old version of Superset?

(Using Postgres connection to ingest the metadata, I get the error: "Cannot convert [LinkedHashMap] due to missing converter implementation." and the connection can't be saved. It's a known issue (#11469).
